target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -71,6 +71,7 @@ ath79_setup_interfaces()
 	tplink,re450-v2|\
 	tplink,re450-v3|\
 	tplink,tl-wr902ac-v1|\
+	tplink,tl-wa1201-v2|\
```

Comment:
Sorting ...

Reply:
What is the sorting logic used? Lexical?

Reply:
Alphabetic, both inside the "blocks" and then for the blocks based on their first entry.